src/lib/util/Config.ts

```diff
@@ -119,6 +130,17 @@ export class Config {
 		return !e || e.disabled;
 	}
 
+	public isDepCheckerEnabled(name: string): boolean {
```

Comment:
The `dev` branch was changed to take an enum for #isEngineEnabled. An enum is more consistent with the other methods such as getSupportedLanguages etc.,